doc/md/predicates.md

```diff
@@ -205,6 +205,42 @@ The above code will produce the following SQL query:
 SELECT DISTINCT `pets`.`id`, `pets`.`owner_id`, `pets`.`name`, `pets`.`age`, `pets`.`species` FROM `pets` WHERE `name` LIKE '_B%'
 ```
 
+#### Escape hatch for `WHERE`
```

Comment:
I'd merge it with the custom predicate section above. Also, the `ExprP` is one way of doing it, you can also achieve the same thing with `P(func(b *sql.Builder) { ... })` - which gives more control over how identifiers and placeholders are formatted. I can suggest another example option for writing it if you like.

Reply:
Like this?

```go
{
	input: Select("id").From(Table("users")).Where(P(func(b *Builder) {
		b.WriteString("DATE(last_login_at) >= ?").Arg("2022-05-03")
	})),
	wantQuery: "SELECT `id` FROM `users` WHERE Date(last_login_at) >= ?",
	wantArgs:  []interface{}{"2022-05-03"},
},
```

Reply:
I think it should be this instead

```go
{
	input: Select("id").
		From(Table("users")).
		Where(P(func(b *Builder) {
			b.WriteString("DATE(").Ident("last_login_at").WriteString(") >= ").Arg("2022-05-03")
		})),
	wantQuery: "SELECT `id` FROM `users` WHERE DATE(`last_login_at`) >= ?",
	wantArgs:  []interface{}{"2022-05-03"},
},
```